domehunter/dome_control.py

```diff
@@ -466,8 +498,12 @@ def _az_to_ticks(self, az):
             Returns encoder tick count corresponding to dome azimuth.
 
         """
+        logger.debug(f'Home azimuth: {self.home_az} Convert Azimuth: {az}')
         az_relative_to_home = (az - self.home_az).wrap_at(360 * u.degree)
-        return az_relative_to_home.degree / self.degrees_per_tick.degree
+        logger.debug(f'Azimuth relative to home: {az_relative_to_home}')
+        encoder_ticks = az_relative_to_home.degree / self.degrees_per_tick.degree
+        logger.debug(f'Encoder ticks for requested az: {encoder_ticks}')
+        return encoder_ticks
```

Comment:
Should this be forced to be an int?

Reply:
I thought I had recommended that in the original PR and there was some reason not to. Although maybe that was something else.

Regardless, I was trying to change as little as possible in this PR other than adding logging. I split it up into a bunch of lines just to log along the way.

But yes, I would think it should be.

Reply:
Oh yea, it should have been fixed in the big pr but I must have forgot... My bad